src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -90,6 +93,8 @@ def _validate_columns(
     :paramtype target: Optional[Callable]
     """
     if target:
+        if any(c.startswith(_OUTPUTS) for c in df.columns):

```

Comment:
What is ther reason for this restriction ?

Reply:
traget will generate columns, which starts with "outputs." and if user will set this column, it will be deleted from the output, however some evaluator may rely on it and the result will be unexpected. Previously, we were mapping to the output from the run and that was not a problem.
It is an edge case, but everything is possible.

Reply:
We may remove the limitation, when we will use pf.run.

Reply:
Instead of fail the call, how about we give user a warning instead?

Reply:
I have marked target outputs as __outputs.